lib/trento/application/event_handlers/stream_roll_up_event_handler.ex

```diff
@@ -13,26 +13,26 @@ defmodule Trento.StreamRollUpEventHandler do
     application: Trento.Commanded,
     name: "stream_roll_up_event_handler"
 
-  alias Trento.Domain.Commands.RollUpCluster
   alias Trento.Hosts.Commands.RollUpHost
   alias Trento.SapSystems.Commands.RollUpSapSystem
+  alias Trento.Clusters.Commands.RollUpCluster
```

Comment:
Credo will complain about alphabetical order I guess

Reply:
indeed it does :smiling_face_with_tear: